pkg/banneduser/banneduser.go

```diff
@@ -0,0 +1,55 @@
+package banneduser
+
+import (
+	"context"
+	"fmt"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const bannedByLabel = toolchainv1alpha1.LabelKeyPrefix + "banned-by"
+
+// NewBannedUser creates a BannedUser resource
```

Comment:
// NewBannedUser creates a BannedUser resource
```suggestion
// NewBannedUser creates a bannedUser resource
```